docs/configuration/index.md

```diff
@@ -1839,4 +1842,4 @@ Supported query contexts:
 |`druid.router.http.readTimeout`|The timeout for data reads from Broker processes.|`PT15M`|
 |`druid.router.http.numMaxThreads`|Maximum number of worker threads to handle HTTP requests and responses|`max(10, ((number of cores * 17) / 16 + 2) + 30)`|
 |`druid.router.http.numRequestsQueued`|Maximum number of requests that may be queued to a destination|`1024`|
-|`druid.router.http.requestBuffersize`|Size of the content buffer for receiving requests. These buffers are only used for active connections that have requests with bodies that will not fit within the header buffer|`8 * 1024`|
\ No newline at end of file
+|`druid.router.http.requestBuffersize`|Size of the content buffer for receiving requests. These buffers are only used for active connections that have requests with bodies that will not fit within the header buffer|`8 * 1024`|
```

Comment:
What has changed in this line?

Reply:
I think my editor deleted the newline at the end of the file. Will fix.

Reply:
I actually have no idea what is up here. The lines are identical so it has to be something to do with a hidden character... The rich diff shows not change for this line so whatever it is, it isn't visible. Still trying to figure it out.